collab-service/src/redis_adapter/redis_pubsub_types.d.ts

```diff
@@ -0,0 +1,13 @@
+declare interface TunnelPubSub<T, V> {
+  registerEvent(
+    call: ServerDuplexStreamImpl<T, V>,
```

Comment:
Once again, Abstract GRPC away. The GRPC library should not be imported at all here.